### DISCUSSION TITLE
Will you ever port into a mod? (preferably fabric)

### DISCUSSION THREAD

**Discussion by basilbvn** [mizarc/bell-claims#121] [Q&A]
Hi! I absolutely adore this plugin, but really wanted to use it on a modded server. It unfortunately doesn't work with cardboard, so i was wondering if you would ever make an official port. My knowledge of coding is pretty limited, so i apologize for asking if this is something that's difficult to do.

**Comment by mizarc** [comment]
I would be a cool idea to build it as a forge/fabric mod, but would require a huge rework of the codebase and likely would implement custom build menu solutions. This would be a far future idea if so.
